src/ui/components/Notice/index.js

```diff
@@ -0,0 +1,67 @@
+/* @flow */
+import makeClassName from 'classnames';
+import * as React from 'react';
```

Comment:
Why not just `import React from 'react';`?

Reply:
The [package](https://reactjs.org/blog/2017/09/26/react-v16.0.html#packaging) changed in React 16 so importing `*` makes more things work as expected.